Missing package `dayjs`, when using one of configuration recipes

**Describe the bug**
When I use this recipe (from https://sbs20.github.io/scanservjs/docs/12-recipes.html):
```js
const dayjs = require('dayjs');
module.exports = {
  afterConfig(config) {
    config.filename = () => {
      return `my_filestem_${dayjs().format('DD-MM-YYYY HH-mm-ss')}`;
    };

    config.log.level = 'DEBUG';
  }
}
```
I have a problem with dependency `dayjs`. I followed guide for upgrading scanservjs from v2 to v3, so I removed npm and other related stuff (I practically just copy-pasted commands from guide on https://sbs20.github.io/scanservjs/docs/01-install.html). After that I copied old `config.local.js` and tried to restart scanservjs (it worked well before), but it failed:

> Nov 14 00:29:45 flat.ceskydj.cz systemd[1]: Stopped scanservjs.
> Nov 14 00:29:45 flat.ceskydj.cz systemd[1]: Started scanservjs.
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: internal/modules/cjs/loader.js:818
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:   throw err;
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:   ^
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: Error: Cannot find module 'dayjs'
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: Require stack:
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: - /etc/scanservjs/config.local.js
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: - /usr/lib/scanservjs/server/classes/user-options.js
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: - /usr/lib/scanservjs/server/application.js
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: - /usr/lib/scanservjs/server/server.js
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Function.Module._load (internal/modules/cjs/loader.js:667:27)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Module.require (internal/modules/cjs/loader.js:887:19)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at require (internal/modules/cjs/helpers.js:74:18)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Object.<anonymous> (/etc/scanservjs/config.local.js:4:15)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Module._compile (internal/modules/cjs/loader.js:999:30)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Module.load (internal/modules/cjs/loader.js:863:32)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Function.Module._load (internal/modules/cjs/loader.js:708:14)
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     at Module.require (internal/modules/cjs/loader.js:887:19) {
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:   code: 'MODULE_NOT_FOUND',
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:   requireStack: [
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     '/etc/scanservjs/config.local.js',
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     '/usr/lib/scanservjs/server/classes/user-options.js',
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     '/usr/lib/scanservjs/server/application.js',
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:     '/usr/lib/scanservjs/server/server.js'
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]:   ]
> Nov 14 00:29:46 flat.ceskydj.cz server.js[3783915]: }
> Nov 14 00:29:46 flat.ceskydj.cz systemd[1]: scanservjs.service: Main process exited, code=exited, status=1/FAILURE
> Nov 14 00:29:46 flat.ceskydj.cz systemd[1]: scanservjs.service: Failed with result 'exit-code'.
> Nov 14 00:29:46 flat.ceskydj.cz systemd[1]: scanservjs.service: Scheduled restart job, restart counter is at 2.
> Nov 14 00:29:46 flat.ceskydj.cz systemd[1]: Stopped scanservjs.
> Nov 14 00:29:46 flat.ceskydj.cz systemd[1]: Started scanservjs.

It looks like the `dayjs` is missing, but I don't know how to resolve this (npm isn't used any more). Could you update the guide, please?

**Server (please complete the following information):**
 - OS: Ubuntu 22.04.3 LTS
 - Node version: v12.22.9
 - NPM version: none

--- a/app-server/config/config.default.js
+++ b/app-server/config/config.default.js
@@ -1,4 +1,7 @@
 /* eslint-disable no-unused-vars */
+const options = { paths: ['/usr/lib/scanservjs'] };
+const Process = require(require.resolve('./server/classes/process', options));
+const dayjs = require(require.resolve('dayjs', options));
 
 /**
  * This file is ignored. If you want to apply overrides, make a copy in this

--- a/docs/01-install.md
+++ b/docs/01-install.md
@@ -32,14 +32,15 @@ and it's a one-off process. Further, you may be using some of the packages that
 this script proposes removing.
 
 ```sh
-# backup scans to home directory
-sudo mv /var/www/scanservjs/data/output ~/scanservjs-scans
-sudo chown $USER:$USER ~/scanservjs-scans
-
 # Stop and remove service
 sudo systemctl stop scanservjs > /dev/null
 sudo rm -vf /etc/systemd/system/scanservjs.service
 sudo systemctl daemon-reload
+
+# backup scans to home directory
+mkdir -p /tmp/scanservjs.bkp
+sudo mv -v /var/www/scanservjs/data /tmp/scanservjs.bkp
+sudo mv -v /var/www/scanservjs/config /tmp/scanservjs.bkp
 
 # Remove all old application files
 sudo rm -rvf /var/www/scanservjs
@@ -57,6 +58,18 @@ sudo apt-get remove -yq \
 
 sudo apt-get autoremove
 ```
+
+To restore backed up files after install:
+
+```sh
+sudo cp -v /tmp/scanservjs.bkp/data/output/* /usr/lib/scanservjs/data/output
+sudo cp -v /tmp/scanservjs.bkp/config/*.local.js /usr/lib/scanservjs/config
+sudo chown $USER:$USER /usr/lib/scanservjs/data/output/*
+```
+
+Note that if you have a `config.local.js` then you may need to amend it to work
+in `v3.x`. Please refer to the updated `config.default.js` or documentation to
+reference the recommended pattern for using node `require(...)`.
 
 ## Arch
 

--- a/docs/10-configuration.md
+++ b/docs/10-configuration.md
@@ -67,7 +67,8 @@ action object must have a name and async execute method taking a `FileInfo`:
 ## Example file
 
 ```javascript
-const Process = require('../server/classes/process');
+const options = { paths: ['/usr/lib/scanservjs'] };
+const Process = require(require.resolve('./server/classes/process', options));
 
 module.exports = {
   /**

--- a/docs/12-recipes.md
+++ b/docs/12-recipes.md
@@ -142,7 +142,8 @@ Then, add the following pipeline:
 ## Change the log level and default scan filename
 
 ```javascript
-const dayjs = require('dayjs');
+const options = { paths: ['/usr/lib/scanservjs'] };
+const dayjs = require(require.resolve('dayjs', options));
 module.exports = {
   afterConfig(config) {
     config.filename = () => {
@@ -258,7 +259,8 @@ integrating with `paperless-ng`. The example below defines a pipeline which
 creates a PDF and then copies it to the home directory on completion.
 
 ```javascript
-const Process = require('../src/classes/process');
+const options = { paths: ['/usr/lib/scanservjs'] };
+const Process = require(require.resolve('./server/classes/process', options));
 
 module.exports = {
   /**
@@ -354,7 +356,8 @@ pipeline then something like the following may help:
 Now create the following pipeline in your `config/config.local.js`
 
 ```javascript
-const Process = require('../server/classes/process');
+const options = { paths: ['/usr/lib/scanservjs'] };
+const Process = require(require.resolve('./server/classes/process', options));
 
 module.exports = {
   afterConfig(config) {

